### PR TITLE
Option to pass strict=True to get_language_from_path. See #18

### PR DESCRIPTION
**Pull request by DrMeers**

### CODE REVIEW

**Comment by st4lk** [comment]
Hi, Simon
Thanks for pull request! Especially thanks for tests!
I'll check it soon.

**Comment by DrMeers** [comment]
No worries, and no rush -- I've found that this still isn't strict enough for my purposes -- /my-slug/ can still activate Burmese! I'll update you once I've figured out the best solution for my project.

File: example/tests/test_solid_urls.py [hunk 1]
```diff
@@ -133,6 +133,23 @@ def test_home_page_default_prefix_ru_404(self):
...
+    def test_about_page_strict_prefix(self):
+        response = self.client.get('/ru/about/')
+        self._base_page_check(response, "ru", "about")
+        self.assertContains(response, '<test>/ru/about/</test>')
+        response = self.client.get('/ru-slug/about/')
+        self._base_page_check(response, "ru", "about")
+        self.assertContains(response, '<test>/ru-slug/about/</test>')
+
+    @override_settings(SOLID_I18N_PREFIX_STRICT=True)
+    def test_about_page_strict_prefix(self):
```
Comment (st4lk) [inline example/tests/test_solid_urls.py]: Unfortunately, this test has the same name as the previous one. As a result, the first test (with `SOLID_I18N_PREFIX_STRICT=False`) will never run.
Propose to use following names:
`test_about_page_strict_prefix_false`
`test_about_page_strict_prefix_true`

And after I've renamed the test, the first one fails (it returns 404).

Comment (DrMeers) [inline example/tests/test_solid_urls.py]: Ha -- thanks! I've fixed the tests, and also made the strict checking more useful which resolves my comment above about the Burmese with "my-slug"... -- see updates below.

**Comment by st4lk** [comment]
Description of problem is similar to #18

**Comment by st4lk** [comment]
Sorry for delay and thanks again!

**Comment by st4lk** [comment]
Now in [pypi v1.3.0](https://pypi.python.org/pypi/solid_i18n/1.3.0)

**Pull request merged**